src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -346,6 +349,10 @@ public void handleRequest(final InputStream inputStream, final OutputStream outp
                 throw new TerminalException("Handler failed to provide a response.");
             }
 
+            if (handlerResponse.getStatus() == OperationStatus.FAILED) {
+                publishExceptionCodeAndCountMetric(request.getAction(), handlerResponse.getErrorCode(), true);
```

Comment:
Here the problem is that if the Operation fails,  only the error code has metric data point as 1. But we are missing the sample data for other error code as 0. Which means, The totally data points is not right.